sig-services/service-requirements/data-collection-permission.md

```diff
@@ -0,0 +1,22 @@
+---
+Title: Data Collection and Permissions
+Persona: service consumer, service provider/owner
+Phase: service-operational
+---
+
+This requirement states that a service should have a well defined social login description. Social login is a single sign-on (SSO) technology that allows users to authenticate themselves on various applications and sites by connecting through social media site rather then typing a separate ID and password on each website.
```

Comment:
```suggestion
If service relies on social login for user identity it should describe why and what information it collects from third party identity providers. Social login is a single sign-on (SSO) technology that allows users to authenticate themselves on various applications and sites by connecting through social media site rather then typing a separate ID and password on each website.
```

Reply:
Thank you @tumido for the feedback. I have made the changes as suggested.